src/type/s2k/generic.js

```diff
@@ -28,16 +28,16 @@
  * @private
  */
 
-import defaultConfig from '../config';
-import crypto from '../crypto';
-import enums from '../enums';
-import util from '../util';
+import defaultConfig from '../../config';
+import crypto from '../../crypto';
+import enums from '../../enums';
+import util from '../../util';
 
 class S2K {
```

Comment:
Please rename this to `GenericS2K`; `S2K` should (if anything) be an abstract base class of `GenericS2K` and `Argon2S2K` (though I don't think it's extremely important to have it)